noxfile.py

```diff
@@ -200,6 +200,18 @@ def lint(session: nox.Session) -> None:
     mypy(session)
 
 
+@nox.session(python="3.11")
```

Comment:
Does this need to be a specific Python version, or can we use any 3.x?

Reply:
If we don't specify a version then nox will run the same python interpreter used to run nox itself. 

I mean, the alternative skip nox (which is for test automation, not for this). 

I can create a python script to create virtualenv , install dependencies, etc instead. then we are not bound by the nox rules.